### PR TITLE
Updated pip_state to work with pip 8.0 

### PR DESCRIPTION
**Pull request by justinta**
Pip 8.0.0 added an InstallationError exception. Updated the state to catch the exception properly.

### CODE REVIEW

**Comment by jtand** [comment]
@jfindlay 


test

File: salt/states/pip_state.py [hunk 1]
```diff
@@ -47,6 +47,12 @@
         if 'pip' in sys.modules:
             del sys.modules['pip']
 
+    pip_ver = tuple(pip.__version__.split('.'))
+    if pip_ver >= (8, 0, 0):
```
Comment (s0undt3ch) [inline salt/states/pip_state.py]: You are comparing a tuple of string to a tuple of ints...

Comment (s0undt3ch) [inline salt/states/pip_state.py]: *strings

Comment (jtand) [inline salt/states/pip_state.py]: fixed. new commit inc.

**Comment by jtand** [comment]
Go Go Jenkins!

**Comment by jtand** [comment]
Go Go Jenkins!

**Pull request merged**